tax_credit/taxa_manipulator.py

```diff
@@ -276,6 +277,7 @@ def stratify_taxonomy_subsets(infile, number_of_splits, basedir,
     # evenly distribute taxa at specified level among chunks
     taxon_dict = dict()
     for taxonomy in line_list:
+        # taxon = delim.join(taxonomy.split(delim)[:level])
```

Comment:
It is generally considered poor form to retain commented-out code in source-controlled code. I believe when Matt discovered that I had done such a thing last Thursday he said "what's this sh*t?".